src/SymptomHistory/SymptomEntryListItem.tsx

```diff
@@ -34,34 +35,48 @@ const SymptomLogListItem: FunctionComponent<SymptomLogListItemProps> = ({
 
   const handleOnPressEdit = () => {
     navigation.navigate(SymptomHistoryStackScreens.SelectSymptoms, {
-      logEntry: JSON.stringify(logEntry),
+      date,
```

Comment:
We may actually want to just pass through the whole entry here and use that data on the SelectSymptoms screen instead of passing through the date and looking up the entry again.